src/test/java/Homework20.java

```diff
@@ -0,0 +1,14 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class Homework20 extends BaseTest {
+
+    public class deletePlaylist() {
+        //Replace thread.sleep with WebDriverWait
+        WebDriverWait wait = WebDriverWait(driver,10);
+        WebElement deleteButton = wait.until(ExpectedConditions.elementToBeClickable(By.cssSelector()));
+    }
+}
```

Comment:
The code is not compiled! Please finish the homework!You got this problem, please correct it.

![image](https://github.com/Test-Pro-Bootcamp/A53-QA-Automation-Framework/assets/104761482/29d7b5dd-7200-4276-9306-890003cc25b0)